toshi_hazard_post/data.py

```diff
@@ -133,7 +131,12 @@ def get_realizations_dataset() -> ds.Dataset:
     """
 
     config = get_config()
-    rlz_dir, filesystem = pyarrow_dataset.configure_output(config['RLZ_DIR'])
+    rlz_dir_tmp = config['RLZ_DIR']
+    if vs30 is not None:
+        rlz_dir_tmp += f"/vs30={vs30}"
```

Comment:
is this folder structure documented, stable, behaviour of pyarrow or internal implementation?

Reply:
as it depends on `toshi-hazard-post` we should discuss with @chrisbc